File: 30 Days of code/src/fr/tenji/java/Days2.java
package fr.tenji.java;

import java.util.Locale;
import java.util.Scanner;

class Days2 {
    public static void main(String[] args) {
//        Scanner scan = new Scanner(System.in).useLocale(Locale.US);
        double mealCost = 12.0; //scan.nextDouble(); // original meal price
        int tipPercent = 20; //scan.nextInt(); // tip percentage
        int taxPercent = 8; //scan.nextInt(); // tax percentage
//        scan.close();

        double tip = mealCost * tipPercent / 100;
        double tax = mealCost * taxPercent / 100;

        int totalCost = (int)Math.round(mealCost + tip + tax);
        System.out.printf("The total meal cost is %d dollars.", totalCost);
    }
}

File: 30 Days of code/src/fr/tenji/js/days2.js
var _input = '';
var input_stdin_array; //eslint-disable no-unused-vars
var count = 0;
/**
 *
 */
function processData() {
    var mealCost = Number(input_stdin_array[0]);
    var tipPercent = Number(input_stdin_array[1]);
    var taxPercent = Number(input_stdin_array[2]);
    var tip = mealCost * tipPercent / 100;
    var tax = mealCost * taxPercent / 100;
    var totalCost = mealCost + tip + tax;
    console.log('The total meal cost is', totalCost.toFixed(0), 'dollars.');
}

process.stdin.resume();
process.stdin.setEncoding('ascii');
// var input = '';
process.stdin.on('data', function (input) {
    _input += input;
    count ++;
    if (count == 3) {
        process.stdin.end();
    }

});
process.stdin.on('close', function () {
    input_stdin_array = _input.split('\n');
    processData(_input);
});
